assets/scss/modules/_images.scss

```diff
@@ -23,54 +22,63 @@
   background-repeat: no-repeat;
 }
 
-.icon-star {
-  @include icon-table;
-  background-image: url('../../img/star.svg');
-}
+.dp-calification {
+  display: flex;
+  align-items: center;
 
-.icon-star-filled {
-  @include icon-table;
-  background-image: url('../../img/star-filled.svg');
+  .icon-star {
+    @include icon-table;
+    color: $dp-color-yellow;
+  }
+
+  .icon-star-filled {
+    @include icon-table;
+    color: $dp-color-silver;
+  }
 }
 
+///
+/// Icons for Subscriber History
+///
+
 // User active
-.icon-user--active:before {
+.user-active:before {
   @include icon-user-table;
   background-image: url('../../img/user-active.svg');
 }
 
 // User active not associated to lists
-.icon-user--natl:before {
+.user-active-with-no-list:before {
   @include icon-user-table;
-  background-image: url('../../img/user-active-not-associated-to-lists.svg');
+  background-image: url('../../img/user-active-with-no-list.svg');
 }
 
 // User removed client
-.icon-user--rc:before {
+.user-removed-client:before {
   @include icon-user-table;
   background-image: url('../../img/user-removed-client.svg');
 }
 
 // User removed hard bounces
-.icon-user--rhb:before {
+.user-removed-hard-bounces:before {
   @include icon-user-table;
   background-image: url('../../img/user-removed-hard-bounces.svg');
 }
 
 // User removed no openings
-.icon-user--rno:before {
+.user-removed-no-openings:before {
   @include icon-user-table;
   background-image: url('../../img/user-removed-no-openings.svg');
 }
 
 // User removed soft bounces
-.icon-user--rsb:before {
+.user-removed-soft-bounces:before {
   @include icon-user-table;
   background-image: url('../../img/user-removed-soft-bounces.svg');
 }
 
 // User subscriber removed
-.icon-user--sr:before {
+.user-subscriber-removed:before {
```

Comment:
How about changing the order here to follow consistency? like `removed-subscriber`